### PR TITLE
Moved articletemplate/variants/world-if into its own repo

### PR DESCRIPTION
**Pull request by sebinsua**
Moved `component-articletemplate/variants/world-if` into its own repo.

### CODE REVIEW

**Comment by sebinsua** [comment]
Please note I will need to [increase the versions once articletemplate is merged and published](https://github.com/economist-components/component-articletemplate/pull/24).

File: header.es6 [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  }
+
+  render() {
+    const { generateClassNameList, mainImage, sectionName, flytitle, title } = this.props;
+
+    let mainImageEl, sectionEl, flytitleEl, titleEl;
+    if (mainImage) {
+      mainImageEl = (
+        <img
+          className={classnames(generateClassNameList(`ArticleTemplate--image`))}
```
Comment (keithamus) [inline header.es6]: same as other variant, `classnames` not needed.

Comment (sebinsua) [inline header.es6]: Okay!

(This repo I obviously spent less time on, because working on world-if felt like it would be an idiotic move...)

**Comment by sebinsua** [comment]
@keithamus Lost the classnames check again.

**Pull request merged**